[2018-12-02 00:00:38,287] [INFO] [localhost][test_appium.py - line:88] app测试开始!
[2018-12-02 00:00:38,343] [INFO] [localhost][test_appium.py - line:54] appium pid is null
[2018-12-02 00:00:38,343] [INFO] [localhost][test_appium.py - line:69] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:00:42,577] [INFO] [localhost][test_appium.py - line:75] 启动appium中...
[2018-12-02 00:00:42,579] [INFO] [localhost][test_appium.py - line:77] appium启动成功
[2018-12-02 00:00:50,167] [INFO] [localhost][test_appium.py - line:91] 启动app中.....
[2018-12-02 00:00:55,173] [INFO] [localhost][test_appium.py - line:124] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:00:55,236] [INFO] [localhost][test_appium.py - line:94] app测试结束!
[2018-12-02 00:04:11,731] [INFO] [localhost][test_appium.py - line:86] app测试开始!
[2018-12-02 00:04:11,790] [INFO] [localhost][test_appium.py - line:52] appium pid is null
[2018-12-02 00:04:11,790] [INFO] [localhost][test_appium.py - line:67] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:04:15,912] [INFO] [localhost][test_appium.py - line:73] 启动appium中...
[2018-12-02 00:04:15,914] [INFO] [localhost][test_appium.py - line:75] appium启动成功
[2018-12-02 00:04:50,247] [INFO] [localhost][test_appium.py - line:86] app测试开始!
[2018-12-02 00:04:50,355] [INFO] [localhost][test_appium.py - line:56] kill appium
[2018-12-02 00:04:50,355] [INFO] [localhost][test_appium.py - line:67] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:04:54,695] [INFO] [localhost][test_appium.py - line:73] 启动appium中...
[2018-12-02 00:04:54,696] [INFO] [localhost][test_appium.py - line:75] appium启动成功
[2018-12-02 00:05:03,395] [INFO] [localhost][test_appium.py - line:89] 启动app中.....
[2018-12-02 00:05:08,400] [INFO] [localhost][test_appium.py - line:122] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:05:08,465] [INFO] [localhost][test_appium.py - line:92] app测试结束!
[2018-12-02 00:07:03,038] [INFO] [localhost][test_appium.py - line:86] app测试开始!
[2018-12-02 00:07:03,198] [INFO] [localhost][test_appium.py - line:56] kill appium
[2018-12-02 00:07:03,199] [INFO] [localhost][test_appium.py - line:67] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:07:07,291] [INFO] [localhost][test_appium.py - line:73] 启动appium中...
[2018-12-02 00:07:07,292] [INFO] [localhost][test_appium.py - line:75] appium启动成功
[2018-12-02 00:07:14,887] [INFO] [localhost][test_appium.py - line:89] 启动app中.....
[2018-12-02 00:07:17,894] [INFO] [localhost][test_appium.py - line:122] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:07:17,951] [INFO] [localhost][test_appium.py - line:92] app测试结束!
[2018-12-02 00:09:33,354] [INFO] [localhost][test_appium.py - line:86] app测试开始!
[2018-12-02 00:09:33,406] [INFO] [localhost][test_appium.py - line:52] appium pid is null
[2018-12-02 00:09:33,406] [INFO] [localhost][test_appium.py - line:67] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:09:37,711] [INFO] [localhost][test_appium.py - line:73] 启动appium中...
[2018-12-02 00:09:37,711] [INFO] [localhost][test_appium.py - line:75] appium启动成功
[2018-12-02 00:09:44,031] [INFO] [localhost][test_appium.py - line:89] 启动app中.....
[2018-12-02 00:09:47,038] [INFO] [localhost][test_appium.py - line:121] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:09:47,096] [INFO] [localhost][test_appium.py - line:92] app测试结束!
[2018-12-02 00:11:56,630] [INFO] [localhost][test_appium.py - line:86] app测试开始!
[2018-12-02 00:11:56,682] [INFO] [localhost][test_appium.py - line:53] appium pid is null
[2018-12-02 00:11:56,682] [INFO] [localhost][test_appium.py - line:68] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:12:00,935] [INFO] [localhost][test_appium.py - line:74] 启动appium中...
[2018-12-02 00:12:00,936] [INFO] [localhost][test_appium.py - line:76] appium启动成功
[2018-12-02 00:12:07,807] [INFO] [localhost][test_appium.py - line:90] 启动app中.....
[2018-12-02 00:12:10,810] [INFO] [localhost][test_appium.py - line:122] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:12:10,868] [INFO] [localhost][test_appium.py - line:93] app测试结束!
[2018-12-02 00:13:33,996] [INFO] [localhost][test_appium.py - line:87] app测试开始!
[2018-12-02 00:13:34,112] [INFO] [localhost][test_appium.py - line:58] kill appium
[2018-12-02 00:13:34,113] [INFO] [localhost][test_appium.py - line:69] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:13:38,253] [INFO] [localhost][test_appium.py - line:75] 启动appium中...
[2018-12-02 00:13:38,253] [INFO] [localhost][test_appium.py - line:77] appium启动成功
[2018-12-02 00:13:45,127] [INFO] [localhost][test_appium.py - line:91] 启动app中.....
[2018-12-02 00:13:48,139] [INFO] [localhost][test_appium.py - line:123] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:13:48,204] [INFO] [localhost][test_appium.py - line:94] app测试结束!
[2018-12-02 00:17:13,598] [INFO] [localhost][test_appium.py - line:87] app测试开始!
[2018-12-02 00:17:13,664] [INFO] [localhost][test_appium.py - line:54] appium pid is null
[2018-12-02 00:17:13,664] [INFO] [localhost][test_appium.py - line:69] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:17:18,077] [INFO] [localhost][test_appium.py - line:75] 启动appium中...
[2018-12-02 00:17:18,078] [INFO] [localhost][test_appium.py - line:77] appium启动成功
[2018-12-02 00:17:25,527] [INFO] [localhost][test_appium.py - line:92] 启动app中.....
[2018-12-02 00:17:28,541] [INFO] [localhost][test_appium.py - line:124] 登录测试异常:'WebDriver' object has no attribute 'driver'
[2018-12-02 00:17:28,543] [INFO] [localhost][test_appium.py - line:95] app测试结束!
[2018-12-02 00:18:26,914] [INFO] [localhost][test_appium.py - line:87] app测试开始!
[2018-12-02 00:18:27,030] [INFO] [localhost][test_appium.py - line:58] kill appium
[2018-12-02 00:18:27,030] [INFO] [localhost][test_appium.py - line:69] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:18:31,176] [INFO] [localhost][test_appium.py - line:75] 启动appium中...
[2018-12-02 00:18:31,176] [INFO] [localhost][test_appium.py - line:77] appium启动成功
[2018-12-02 00:18:39,476] [INFO] [localhost][test_appium.py - line:91] 启动app中.....
[2018-12-02 00:18:42,568] [INFO] [localhost][test_appium.py - line:94] app测试结束!
[2018-12-02 00:19:18,540] [INFO] [localhost][test_appium.py - line:87] app测试开始!
[2018-12-02 00:19:18,649] [INFO] [localhost][test_appium.py - line:58] kill appium
[2018-12-02 00:19:18,649] [INFO] [localhost][test_appium.py - line:69] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:19:22,821] [INFO] [localhost][test_appium.py - line:75] 启动appium中...
[2018-12-02 00:19:22,823] [INFO] [localhost][test_appium.py - line:77] appium启动成功
[2018-12-02 00:19:30,133] [INFO] [localhost][test_appium.py - line:91] 启动app中.....
[2018-12-02 00:19:33,807] [INFO] [localhost][test_appium.py - line:117] 点击授权
[2018-12-02 00:19:35,317] [INFO] [localhost][test_appium.py - line:117] 点击授权
[2018-12-02 00:19:41,649] [INFO] [localhost][test_appium.py - line:94] app测试结束!
[2018-12-02 00:31:57,626] [INFO] [localhost][test_appium.py - line:87] app测试开始!
[2018-12-02 00:31:57,688] [INFO] [localhost][test_appium.py - line:54] appium pid is null
[2018-12-02 00:31:57,689] [INFO] [localhost][test_appium.py - line:69] appium启动命令:appium --log ./appium.log --session-override --udid 192.168.56.101:5555 -a  127.0.0.1 -p 4723
[2018-12-02 00:32:02,756] [INFO] [localhost][test_appium.py - line:75] 启动appium中...
[2018-12-02 00:32:02,759] [INFO] [localhost][test_appium.py - line:77] appium启动成功
[2018-12-02 00:32:12,467] [INFO] [localhost][test_appium.py - line:91] 启动app中.....
[2018-12-02 00:32:18,487] [INFO] [localhost][test_appium.py - line:94] app测试结束!
